playbooks/adhoc/docker_storage_to_overlay2/docker_storage_to_overlay2.yml

```diff
@@ -0,0 +1,195 @@
+---
+# This playbook changes the Docker storage driver to overlay2.
+#
+# It requires support for the "overlay2.size" daemon storage option to
+# preserve the devicemapper disk quota ("dm.basesize").  This feature
+# is beyond Docker 1.12.  See:
+#
+#    https://github.com/moby/moby/pull/24771
+#
+#    and
+#
+#    https://github.com/moby/moby/pull/32977
+#
+# Note, this makes no attempt to migrate existing containers/images.
+# It momentarily stops docker.service and uses "atomic storage reset"
+# to wipe all local Docker storage.
+#
+# To run:
+#
+# ansible-playbook -e 'cli_tag_name=<tag-name>' docker_storage_to_overlay2.yml
+#
+
+- name: Transition Docker storage driver to overlay2
+  hosts: nodes
```

Comment:
Just checking -- this will also update the driver on the masters, right? Ops would like the storage driver configurations to be consistent across masters and nodes.

Reply:
@dak1n1 Ah okay, wasn't clear to me what to use for `hosts`.

Should this then be:
```
hosts:
- masters
- nodes
```
?